### PR TITLE
[cleanup]: remove unused functions and variables

### PR DESCRIPTION
**Pull request by iholder101**
### What this PR does
Removes unused functions and variables.

/kind cleanup

### Release note

```release-note
NONE
```

### CODE REVIEW

**Review by ShellyKa13 — COMMENTED** [review]
Best kind of PRS!
/lgtm

**Review by ShellyKa13 — APPROVED** [review]

File: tests/storage/datavolume.go [hunk 1]
```diff
@@ -1494,30 +1494,6 @@ func volumeExpansionAllowed(sc string) bool {
 		*storageClass.AllowVolumeExpansion
 }
 
-func newRandomVMWithDataVolume() (*v1.VirtualMachine, bool) {
```
Comment (orelmisan) [inline tests/storage/datavolume.go:1497]: I couldn't find this function on latest `main` branch.
Was it already removed?

**Review by EdDev — APPROVED** [review]
Approving the network codebase:

/approve

File: pkg/virt-api/webhooks/arm64.go [hunk 2]
```diff
@@ -27,12 +27,6 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 )
 
-var _false bool = false
```
Comment (orelmisan) [inline pkg/virt-api/webhooks/arm64.go:30]: I see one usage of it in `setDefaultArm64Bootloader`.

File: pkg/virt-api/webhooks/arm64.go [hunk 3]
```diff
@@ -27,12 +27,6 @@ import (
...
-
-const (
-	defaultCPUModelArm64 = v1.CPUModeHostPassthrough
```
Comment (orelmisan) [inline pkg/virt-api/webhooks/arm64.go:33]: I see one usage of it in `setDefaultArm64CPUModel`.

**Review by orelmisan — COMMENTED** [review]
Thank you for the PR @iholder101.
I see usages of the variable / constant that were removed on the last commit.
Could you please have a look?

File: pkg/virt-api/webhooks/arm64.go (hunk 3 above)
Comment (orelmisan) [reply pkg/virt-api/webhooks/arm64.go:33]: I was mistakenly looking at `pkg/defaults/arm64.go` that looks very similar.
Sorry for the noise.

File: pkg/virt-api/webhooks/arm64.go (hunk 2 above)
Comment (orelmisan) [reply pkg/virt-api/webhooks/arm64.go:30]: I was mistakenly looking at `pkg/defaults/arm64.go` that looks very similar.
Sorry for the noise.

File: tests/storage/datavolume.go (hunk 1 above)
Comment (orelmisan) [reply tests/storage/datavolume.go:1497]: It was removed by #13268.

**Comment by iholder101** [comment]
Since three people have approved this small PR, two of them being sig approvers, and since the PR is tiny and straightforward, I'll take the liberty of approving it.

/approve

**Pull request merged**